spec/PeriodicBackgroundSync-index.bs

```diff
@@ -93,3 +93,28 @@ The <dfn>tag</dfn> is a DOMString. Within one [=list of periodicsync registratio
 The options are a dictionary containing [=minInterval=], a long long. <dfn>minInterval</dfn> is used to specify the minimum interval, in milliseconds, at which the periodic synchronization should happen.
 
 The <dfn>registration state</dfn> is one of <dfn>pending</dfn>, <dfn>waiting</dfn>, <dfn>firing</dfn>, or <dfn>reregisteredWhileFiring</dfn>. It is initially set to [=pending=].
+
+
+Privacy Considerations {#privacy}
+==================================
+
+## Permission ## {#permission}
+Periodic Background Sync is only available if the user (or user agent on behalf of the user) has granted <code>periodic-background-sync</code> permission. The permission to schedule periodic syncs for a given origin is one of three strings:
+
+<code>"default"</code>
+This is equivalent to <code>"denied"</code>, but the user has made no explicit choice thus far.
+
+<code>"denied"</code>
+This means the user does not want periodic sync for the origin.
+
+<code>"granted"</code>
+This means periodic syncing is allowed for the origin.
+
+In addition, user agents MAY offer a way for the user to disable periodic background sync.
+
+## Location Tracking ## {#location-tracking}
+Fetch requests within the periodicsync event while [=in the background=] may reveal the client's IP address to the server after the user has left the page. The user agent SHOULD limit tracking by capping the number of retries and duration of periodicsync events.
+Further, the user agent SHOULD limit persistent location tracking by capping the frequency of periodicsync events, both for an origin, and across origins.
+
+## History Leaking ## {#history-leaking}
+Fetch requests within the periodicsync event while [=in the background=] may reveal something about the client's navigation history to passive eavesdroppers. For instance, the client might visit site https://example.com, which registers a periodicsync event, but doesn't fire until after the user has navigated away from the page and changed networks. Passive eavesdroppers on the new network may see the fetch requests that the periodicsync event makes. The fetch requests are HTTPS so the request contents will not be leaked but the domain may be (via DNS lookups and IP address of the request).
```

Comment:
You could add a SHOULD or MAY statement here about limiting PBS to the network the registration was created on, and what the trade-offs for that are.